plio/signals.py

```diff
@@ -28,3 +28,11 @@ def item_update_cache(sender, instance, **kwargs):
 def question_update_cache(sender, instance, **kwargs):
     # invalidate saved cache for the plio
     invalidate_cache_for_instance(instance.item.plio)
+
+
+@receiver([post_save], sender=Question)
```

Comment:
Why not post_delete?

Reply:
haan I thought so at first too. turns out that because we are doing soft_delete, `post_delete` is never invoked only.